ts/features/bonus/cgn/saga/networking/bucket/index.ts

```diff
@@ -0,0 +1,12 @@
+import { delay, put } from "redux-saga/effects";
+import { cgnCodeFromBucket } from "../../../store/actions/bucket";
+
+// handle the request for CGN Otp code generation
```

Comment:
```suggestion
// handle the request for CGN bucket consumption
```
Maybe from a copy/paste?